demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c

```diff
@@ -627,33 +627,22 @@ CK_RV xProvisionCertificate( CK_SESSION_HANDLE xSession,
     return xResult;
 }
 
-CK_RV xDestroyCredentials( CK_SESSION_HANDLE xSession )
+CK_RV xDestroyProvidedCredentials( CK_SESSION_HANDLE xSession,
```

Comment:
I don't think that credentials is the right term. For one thing, the only type of credential we directly support with this code is X.509 certificates and private keys. For another thing, this helper routine is sufficiently abstract that it could be used to delete any type of object by label + class, including those that may not be used for authentication.

Reply:
I will rename this to xDestroyProvidedObjects.